packages/edit-patterns/src/components/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import PatternConvertButton from './pattern-convert-button';
+import PatternsManageButton from './patterns-manage-button';
+
+export default function PatternsMenuItems( { rootClientId } ) {
```

Comment:
It's a bit weird to me that this component is basically expected to be rendered within a `BlockEditorProvider` to actually work properly. 

Do you think there's a possibility to remove the dependency to `block-editor` entirely to make the component more reusable?

If we are to keep it as it is, we should clarify that this package is not about "patterns" but about "using patterns within a block editor" or something like that.

Reply:
Thanks for the feedback @youknowriad. Currently, it is just a copy of how the equivalent [reusable block component](https://github.com/WordPress/gutenberg/blob/trunk/packages/reusable-blocks/src/components/reusable-blocks-menu-items/index.js#L13) that it is deprecating was set up.

It is a bit hard to see how this particular component could be reusable outside an editor context as it augments the editor block menus, and needs access to currently selected blocks, etc., which could be passed in as props instead of deps, but hard to see what they would be swapped out for in a non-editor context.

I did initially just add these components to the `editor` package, but moved it to a new package after [this comment](https://github.com/WordPress/gutenberg/pull/53161#issuecomment-1658072417). 

I am open to ideas about where these new components that deprecate the reusable block ones can best be located.

Reply:
about the "reusable-blocks" package. I'm not a big fan of that one much either but I think it's probably "ok", let's just document that this is a block-editor specific component.

Reply:
Thanks for the additional context. I have updated the [package readme](https://github.com/WordPress/gutenberg/pull/53161/files#diff-87865f128cc1dc55d61ae27ec3600a115663235a8aa57d6ef90bc7bbaec6109fR15). There is no urgency for this merge so will keep it open for a few days, soget back to me if you have any additional thoughts about a better location for these components.